tests/StatsdClient.Tests/GlobalSetup.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using NUnit.Framework;
+using StatsdClient;
+
+[SetUpFixture]
+public class GlobalSetup
```

Comment:
Is `GlobalSetup` needed as setting the env var is done in a single function `Setting_tag_with_env_arg`?
Note: If `Environment.GetEnvironmentVariable` starts to be annoying in the unit tests, I will mock `Environment.GetEnvironmentVariable`.

Reply:
This is not strictly needed, but remember that the statsd client now automatically populates tags from environment variables. On my machine I have the DD_ENV variable set, and so completely unrelated tests started failing because of the unexpected extra tag. The GlobalSetup is not needed for the CI, but this is a convenience for developers running tests locally.